python/common/org/python/types/Date.java

```diff
@@ -0,0 +1,104 @@
+package org.python.types;
+
+import java.time.YearMonth;
+
+public class Date extends org.python.types.Object {
+    public final static int MIN_YEAR = 1;
+    public final static int MAX_YEAR = 9999;
+    public final static int MIN_MONTH = 0;
+    public final static int MAX_MONTH = 11;
+    public final static int MIN_DAY = 1;
+    public final static int MAX_DAY = 31;
+
+    @org.python.Attribute()
+    public final static Date min = new Date(MIN_YEAR, MIN_MONTH, MIN_DAY);
+    @org.python.Attribute()
+    public final static Date max = new Date(MAX_YEAR, MAX_MONTH, MAX_DAY);
+    @org.python.Attribute()
+    public final static Date resolution = null; // TODO: implement `resolution` when timedelta is implemented
```

Comment:
Will you ignore this for now, or will you fix it after #34 is merged?

Reply:
I think the plan is to wait with it for now :)